Orange/modelling/knn.py

```diff
@@ -0,0 +1,10 @@
+from Orange.classification import KNNLearner as KNNClassification
+from Orange.modelling.fitter import Fitter, LearnerTypes
+from Orange.regression import KNNRegressionLearner
+
+
+class KNNLearner(Fitter):
+    name = 'knn'
+
+    __fits__ = LearnerTypes(classification=KNNClassification,
+                            regression=KNNRegressionLearner)
```

Comment:
Yes, I think most of them will be fairly short, with the possible exception of AdaBoost and perhaps SVM and LR. I'm fairly certain that further changes will need to be made to the Fitter, but the current implementation should cover at least half of the learners in Orange atm.